recorder/src/main/cpp/stacktraces.cc

```diff
@@ -0,0 +1,23 @@
+#include "stacktraces.hh"
+
+std::uint32_t Stacktraces::fill_backtrace(NativeFrame* buff, std::uint32_t capacity) {//TODO: write me 3 tests { (capacity > stack), (stack > capacity) and (stack == capacity) }
+    std::uint64_t rbp, rpc;
+    asm("movq %%rbp, %%rax;"
+        "movq %%rax, %0;"
+        "lea (%%rip), %%rax;"
```

Comment:
I was talking about using %rbx as general register not %rbp. But ya I got the answer.